polaris-service/src/main/java/org/apache/polaris/service/config/PolarisApplicationConfig.java

```diff
@@ -151,12 +151,13 @@ public void setFeatureConfiguration(Map<String, Object> featureConfiguration) {
 
   @JsonProperty("maxRequestBodyBytes")
   public void setMaxRequestBodyBytes(long maxRequestBodyBytes) {
-    if (maxRequestBodyBytes == 0) {
-      // The underlying library that we use to implement the limit treats -1 and 0 as the same, so
-      // we block 0 to prevent ambiguity.
+    if (maxRequestBodyBytes <= 0 && maxRequestBodyBytes != -1) {
+      // The underlying library that we use to implement the limit treats all values <= 0 as the
+      // same, so
+      // we block all but -1 to prevent ambiguity.
```

Comment:
nit: This formatting looks a bit odd. Can it be 2 lines?

Reply:
Updated.

I've just been letting spotless take the wheel with these, but I guess it's ok with either style? We should probably update it to prefer a single format if possible.